Shared/mods/deathmatch/logic/luadefs/CLuaCryptDefs.cpp

```diff
@@ -402,10 +402,17 @@ int CLuaCryptDefs::EncodeString(lua_State* luaVM)
     CScriptArgReader argStream(luaVM);
     argStream.ReadEnumString(algorithm);
     argStream.ReadString(data);
-    argStream.ReadStringMap(options);
 
-    argStream.ReadFunction(luaFunctionRef, LUA_REFNIL);
-    argStream.ReadFunctionComplete();
+    if ((algorithm != StringEncodeFunction::BASE64 && algorithm != StringEncodeFunction::BASE32) || argStream.NextIsTable())
```

Comment:
This makes `base64` and `base32` algorithms syntactically inconsistent with the older ones. For example, you can omit the `options` argument for new algorithms, while must preserve it for, let's say, `tea`. This is confusing.

Reply:
There was some conversation on Discord about this, @botder said its fine.
https://discord.com/channels/801330706252038164/801411628600000522/1206737515940155534
https://discord.com/channels/801330706252038164/801411628600000522/1206757450162774079

Reply:
What do you think, it is acceptable in this case? Or do you have any better ideas?

Reply:
Can be marked as solved, according to the last conversation in Discord. https://discord.com/channels/801330706252038164/801411628600000522/1208271751214276639